loopchain/launcher.py

```diff
@@ -108,28 +108,35 @@ def start_as_rest_server(args):
     amqp_key = args.amqp_key or conf.AMQP_KEY
     api_port = int(peer_port) + conf.PORT_DIFF_REST_SERVICE_CONTAINER
     conf_path = conf.CONF_PATH_ICONRPCSERVER_DEV
+    dev_mode = True
 
     if args.radio_station_target:
         if args.radio_station_target == conf.URL_CITIZEN_TESTNET:
             conf_path = conf.CONF_PATH_ICONRPCSERVER_TESTNET
+            dev_mode = False
         elif args.radio_station_target == conf.URL_CITIZEN_MAINNET:
             conf_path = conf.CONF_PATH_ICONRPCSERVER_MAINNET
+            dev_mode = False
 
     from iconrpcserver.default_conf.icon_rpcserver_config import default_rpcserver_config
     from iconrpcserver.icon_rpcserver_cli import start_process, find_procs_by_params
     from iconcommons.icon_config import IconConfig
 
     additional_conf = {
         "config": conf_path,
-        "channel": channel,
-        "port": api_port,
-        "amqpKey": amqp_key,
         "tbearsMode": False
     }
 
-    rpcserver_conf = IconConfig("", default_rpcserver_config)
+    rpcserver_conf: IconConfig = IconConfig("", default_rpcserver_config)
     rpcserver_conf.load()
     rpcserver_conf.update_conf(additional_conf)
+    if dev_mode:
+        additional_conf.update({
+            "channel": channel,
+            "port": api_port,
+            "amqpKey": amqp_key
+        })
+        rpcserver_conf.update_conf(additional_conf)
```

Comment:
It would be better `additional_conf.update` place on above create instance of `IconConfig` line.
then, `rpcserver_conf.update_conf` will be called once when dev_mode.

Reply:
As you mentioned, I removed the part that calls update_conf (additional_conf) twice. Thank you.